v2/bundle/spiffebundle/bundle.go

```diff
@@ -152,127 +152,127 @@ func (b *Bundle) TrustDomain() spiffeid.TrustDomain {
+// AddX509Authority adds an X.509 authority to the bundle. If the authority already
 // exists in the bundle, the contents of the bundle will remain unchanged.
-func (b *Bundle) AddX509Root(x509Root *x509.Certificate) {
+func (b *Bundle) AddX509Authority(x509Authority *x509.Certificate) {
 	b.mtx.Lock()
 	defer b.mtx.Unlock()
 
-	for _, r := range b.x509Roots {
-		if r.Equal(x509Root) {
+	for _, r := range b.x509Authorities {
+		if r.Equal(x509Authority) {
 			return
 		}
 	}
 
-	b.x509Roots = append(b.x509Roots, x509Root)
+	b.x509Authorities = append(b.x509Authorities, x509Authority)
 }
 
-// RemoveX509Root removes an X.509 root from the bundle.
-func (b *Bundle) RemoveX509Root(x509Root *x509.Certificate) {
+// RemoveX509Authority removes an X.509 authority from the bundle.
+func (b *Bundle) RemoveX509Authority(x509Authority *x509.Certificate) {
 	b.mtx.Lock()
 	defer b.mtx.Unlock()
 
-	for i, r := range b.x509Roots {
-		if r.Equal(x509Root) {
-			b.x509Roots = append(b.x509Roots[:i], b.x509Roots[i+1:]...)
+	for i, r := range b.x509Authorities {
+		if r.Equal(x509Authority) {
+			b.x509Authorities = append(b.x509Authorities[:i], b.x509Authorities[i+1:]...)
 			return
 		}
 	}
 }
 
-// HasX509Root checks if the given X.509 root exists in the bundle.
-func (b *Bundle) HasX509Root(root *x509.Certificate) bool {
+// HasX509Authority checks if the given X.509 authority exists in the bundle.
+func (b *Bundle) HasX509Authority(x509Authority *x509.Certificate) bool {
 	b.mtx.RLock()
 	defer b.mtx.RUnlock()
 
-	for _, r := range b.x509Roots {
-		if r.Equal(root) {
+	for _, r := range b.x509Authorities {
+		if r.Equal(x509Authority) {
 			return true
 		}
 	}
 	return false
 }
 
-// SetX509Roots sets the X.509 roots in the bundle.
-func (b *Bundle) SetX509Roots(roots []*x509.Certificate) {
+// SetX509Authorities sets the X.509 authorities in the bundle.
+func (b *Bundle) SetX509Authorities(authorities []*x509.Certificate) {
 	b.mtx.Lock()
 	defer b.mtx.Unlock()
 
-	b.x509Roots = x509util.CopyX509Roots(roots)
+	b.x509Authorities = x509util.CopyX509Authorities(authorities)
 }
 
-// JWTKeys returns the JWT keys in the bundle, keyed by key ID.
-func (b *Bundle) JWTKeys() map[string]crypto.PublicKey {
+// JWTAuthorities returns the JWT authorities in the bundle, keyed by authority ID.
+func (b *Bundle) JWTAuthorities() map[string]crypto.PublicKey {
 	b.mtx.RLock()
 	defer b.mtx.RUnlock()
 
-	return jwtutil.CopyJWTKeys(b.jwtKeys)
+	return jwtutil.CopyJWTAuthorities(b.jwtAuthorities)
 }
 
-// FindJWTKey finds the JWT key with the given key id from the bundle. If the key
+// FindJWTAuthorities finds the JWT authority with the given authority id from the bundle. If the authority
 // is found, it is returned and the boolean is true. Otherwise, the returned
 // value is nil and the boolean is false.
-func (b *Bundle) FindJWTKey(keyID string) (crypto.PublicKey, bool) {
+func (b *Bundle) FindJWTAuthorities(authorityID string) (crypto.PublicKey, bool) {
```

Comment:
Since a single authority is returned, this should be expressed in singular, not plural.